### PR TITLE
add transition and fade components to alert component

### PR DESCRIPTION
**Pull request by hafabenny**
struggling with adding Fade component to project.
Can't figure out how to build a simple fade example story
Added Fade component to Alert component.  same problem as Fade simple story.

I believe there are two main CSS classes related to Fade component:
- fade
- show

the show CSS class is defined like

.fade.show

how do we make sure CSS to JS worked for the .fade and .show CSS selectors?

### CODE REVIEW

**Comment by bwarner** [comment]
ok I figured it out, don't combine show with the other styles, I'm using 
```
      className={`${classes} ${context.isOpen && show}`}
```
in the DropdownMenu class.

**Comment by bwarner** [comment]
I've add pick and pluck from lodash so we don't need the implementations in utilities

**Comment by hafabenny** [comment]
need to fix conflicts
need to get dismissible alert working. how to pass state flag from stories to component?

**Comment by hafabenny** [comment]
fixed issue with dismiss story and state management, used lodash.omit and pick.  fixed issue with fade and show classes.  thanks for all the help.

File: stories/fade.stories.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Button } from '../lib/button';
+import Fade from '../lib/fade';
+
+// constructor(props) {
```
Comment (bwarner) [inline stories/fade.stories.js]: No need for unused commented code

File: stories/index.stories.js [hunk 2]
```diff
@@ -68,4 +72,4 @@ storiesOf('Pagination', module)
   .add('Pagination Disabled', PaginationDisabledStory)
   .add('Pagination Large', PaginationLargeStory)
   .add('Pagination Small', PaginationSmallStory)
-  .addDecorator(withReadme(PaginationReadme));
+  .addDecorator(withReadme(PaginationReadme));
```
Comment (bwarner) [inline stories/index.stories.js]: Add newline to end of file.

**Pull request merged**